Podfile

```diff
@@ -8,31 +8,31 @@ target 'AlphaWallet' do
   pod 'R.swift'
   pod 'JSONRPCKit', '~> 2.0.0'
   pod 'APIKit'
-  pod 'Eureka', '~> 5.2.1'
+  pod 'Eureka', :git=> 'https://github.com/xmartlabs/Eureka.git', :branch => 'xcode12'
   pod 'MBProgressHUD'
   pod 'StatefulViewController'
 
   pod 'QRCodeReaderViewController', :git=>'https://github.com/AlphaWallet/QRCodeReaderViewController.git', :commit=>'30d1a2a7d167d0d207ae0ae3a4d81bcf473d7a65'
-  pod 'KeychainSwift', :git=>'https://github.com/AlphaWallet/keychain-swift.git', :branch=>'alphawallet'
+  pod 'KeychainSwift', :git=>'https://github.com/AlphaWallet/keychain-swift.git', :commit=> 'alphawallet'
```

Comment:
Must have missed this earlier. Can you make it a hash?

Reply:
yep, i missed this, just a sec